packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -61,16 +61,39 @@ const classes = {
   )
 }
 
+type ActiveRoute = {
+  active: boolean
+  activeRouteInside: boolean
+}
+
+function useActiveRoute(itemRoute: string | undefined): ActiveRoute {
```

Comment:
This part isn't really necessary, just an opportunity to share some logic in this file :)